nike_bot.py

```diff
@@ -1,63 +1,63 @@
-if __name__ == "__main__":
-	import requests
-	import time
-	import twitter
+#if __name__ == "__main__":
+#	import requests
+#	import time
+#	import twitter
 
 	# setup your own account and get these values from twitter
-	api = twitter.Api(consumer_key="CONSUMER_KEY_HERE",
-		consumer_secret="CONSUMER_SECRET_HERE",
-		access_token_key="ACCESS_TOKEN_HERE",
-		access_token_secret="ACCESS_TOKEN_SECRET")
+#	api = twitter.Api(consumer_key="CONSUMER_KEY_HERE",
+#		consumer_secret="CONSUMER_SECRET_HERE",
+#		access_token_key="ACCESS_TOKEN_HERE",
+#		access_token_secret="ACCESS_TOKEN_SECRET")

```

Comment:
this is for connecting to twitter, you will need a twitter developer account to do this, which just requires a sign up